tartiflette/language/validators/__init__.py

```diff
@@ -0,0 +1,16 @@
+class Validators:
+
+    __slots__ = ("ctx", "schema", "errors", "rules")
+
+    def __init__(self, schema, rules):
```

Comment:
If agree with the `ValidationRule` base class, we could have a signature like this one:
```suggestion
    def __init__(self, schema: "GraphQLSchema", rules: List["ValidationRule"]) -> None:
```